tests/test_export_engine.py

```
@@ -0,0 +1,94 @@
+    def test_nocache_env_passed_to_prepare_workspace_orchestration(self):
+        with patch.dict(os.environ, {"CURSOR_CHAT_BROWSER_NOCACHE": "1"}):
+            with patch(
+                "services.export_engine.prepare_workspace_orchestration",
+            ) as mock_prepare:
+                mock_prepare.return_value = MagicMock(spec=WorkspaceOrchestration)
+                with patch(
+                    "services.export_engine.load_global_db_export_data",
+                    return_value=None,
+                ):
+                    collect_export_entries(
+                        workspace_path="/tmp/ws",
```

Ruff (0.15.18)

[error] Probable insecure usage of temporary file or directory: "/tmp/ws"

(S108)